scripts/migrate-unknown.md

```diff
@@ -0,0 +1,39 @@
+# Migrate Unknown Patients into Restricted Patients
+
+## Overview
+
+We have many patients who do not have NOMIS records but need adding to Restricted Patients so that they are available to MPC.
+
+## Pre-requisites
+
+Obtain a copy of the spreadsheet containing these patients - currently named `15922 MHCS RP data no queries`. Export the worksheet `MHCS data included` as a CSV file and copy it into this directory with file name `unknown-patients.csv`.
```

Comment:
didn't you have to re-format the dates of birth so that they could be imported easily?